### PR TITLE
Week9

### PR DESCRIPTION
**Pull request by aditya226369**
@anilrao316
### Assignment - 9:
Creating Todo List Interface in reactjs.

### CODE REVIEW

**Comment by anilrao316** [comment]
Please attaching recordimg of working solution

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 **/.vscode/*
-**/node_modules/*
```
Comment (anilrao316) [inline .gitignore:2]: Remove this file

Comment (anilrao316) [inline .gitignore:2]: Remove this file

File: Week-9/todolist/src/Components/Todoform.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    const handleChange=e=>{
+        setInput(e.target.value);
+    }
+
+    const handleSubmit = e=>{
+        e.preventDefault();
+        if(input===''){
+            return;
+        }
+        props.onSubmit({
```
Comment (anilrao316) [inline Week-9/todolist/src/Components/Todoform.js:15]: Matbe u can rename this component to addTodo.. even the file

File: Week-9/todolist/src/Components/Todo.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+function Todo({ todos, completeTodo, removeTodo }) {
+  
+  return todos.map((todo, index) => (
+    <div className="todo_row" key={index}>
+      <div key={todo.id}>{todo.text}</div>
+      <div className="icons">
+      <div className={todo.isComplete ? "status_text_complete" : "status_text_pending"} /* this tells wheather the state is pending or completed */ >
+        {todo.isComplete ? "Completed" : "Pending"} 
+        <input className="checkBox" type="checkbox" onClick={()=>completeTodo(todo.id)} /* this toggles the state between pending or completed */ />
```
Comment (anilrao316) [inline Week-9/todolist/src/Components/Todo.js]: If possible add unit testcases for ur learning

**Comment by aditya226369** [comment]
@anilrao316 
Please find the video of code overview
https://www.loom.com/share/b00122ef16c64ec3a49c63c816f7dc9a

**Pull request merged**